dcpy/utils/data.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
+import geopandas as gpd
+
+from pathlib import Path
+
+from dcpy.utils.logging import logger
+from dcpy.models import file
+from dcpy.models.lifecycle.ingest import (
+    Config,
+)
+
+import zipfile
+
+EXTACTED_FILES_DIR = Path("extracted_files")
```

Comment:
Maybe `tmp/extracted_files`? Just so it's gitignored/hidden slightly more.

Also, typo in var name!